containers/Makefile

```diff
@@ -112,8 +112,14 @@ build/%: %
 	cp --archive --parents "$<" build/
 
 # Templates
-build/%: %.j2 config.yml ../tools/render-jinja-template.sh
-	../tools/render-jinja-template.sh config.yml "$<" "$@"
+build/%: %.j2 build/vars-merged-with-config.yml ../tools/render-jinja-template.sh
+	../tools/render-jinja-template.sh build/vars-merged-with-config.yml "$<" "$@"
```

Comment:
Please break this command into multiple lines because it's starting to get hard to read.

Reply:
Done.